config/test.yml.erb

```diff
@@ -21,6 +21,7 @@ dashboard_devise_pepper: not a pepper!
 dashboard_secret_key_base: not a secret
 google_maps_api_key: boguskey
 db_writer: 'mysql://root@localhost/'
+firebase_shared_secret: fake_firebase_shared_secret
```

Comment:
is this needed to make drone pass? this looks like it will override the secret pulled in by config.yml.erb, because per-environment config files like this one override the global one (logic for that here https://github.com/code-dot-org/code-dot-org/blob/442f0f2f8f3a8a021f3ac3767e612f9850a9239a/lib/cdo.rb#L18-L36), which could make this setting unavailable to any UI tests on the test machine (and probably also on drone) . If this is just to get unit tests to pass on drone, you might try providing this secret via drone instead. I don't know how to do this, but AFAIK we supply other secrets to drone runs via the drone web UI.

Reply:
`test.yml.erb` also conditionally has `clear_secrets`, for both the unit test and drone cases, so I think this is fine. See note in https://github.com/code-dot-org/code-dot-org/blob/staging/config/secrets.md about `clear_secrets`.